backend/de.metas.acct.base/src/main/sql/postgresql/ddl/views/c_bpartner_adv_search_v.sql

```diff
@@ -1,23 +1,24 @@
-DROP VIEW IF EXISTS C_BPartner_Adv_Search_v;
+DROP VIEW IF EXISTS C_BPartner_Adv_Search_v
+;
+
 CREATE OR REPLACE VIEW C_BPartner_Adv_Search_v AS
 SELECT p.c_bpartner_id,
        pl.c_bpartner_location_id,
+       COALESCE(u.ad_user_id,-1) as userID,
```

Comment:
use AD_User_ID

Reply:
As discussed changed to `C_BP_Contact_ID`